lib/crystalball/predictor.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'crystalball/predictor/modified_execution_paths'
 require 'crystalball/predictor/modified_specs'
+require 'crystalball/predictor/direct_map'
```

Comment:
predictor class by itself doesn't depend on any of strategies required above. Let's move those requires to the place where we use them

Reply:
It feels like these predictors should be available after `require 'crystalball'`

Reply:
done